source/common/network/apple_dns_impl.cc

```diff
@@ -55,7 +55,7 @@ AppleDnsResolverStats AppleDnsResolverImpl::generateAppleDnsResolverStats(Stats:
 AppleDnsResolverImpl::StartResolutionResult
 AppleDnsResolverImpl::startResolution(const std::string& dns_name,
                                       DnsLookupFamily dns_lookup_family, ResolveCb callback) {
-  ENVOY_LOG(debug, "DNS resolver resolve={}", dns_name);
+  ENVOY_LOG_EVENT(debug, "apple_dns_started", "DNS resolution for {} started", dns_name);
```

Comment:
```suggestion
  ENVOY_LOG_EVENT(debug, "apple_dns_start", "DNS resolution for {} started", dns_name);
```
To keep it a noun (as with other cases in this file)